receiver/azuremonitorreceiver/testdata/expected_metrics/metrics_golden.yaml

```diff
@@ -1,9 +1,6 @@
 resourceMetrics:
```

Comment:
expected_metrics diff is not readable as I sorted by metrics name with ``yq``, specifically to have a readable diff for me.
> TODO(celian): Add a readable diff here with commands executed.